rosie_map_controller/WallDefinition[] NewWalls

